### ISSUE TITLE
TextBox border and floating hint colors don't follow Material Design guidelines

### ISSUE THREAD

**Issue by redcurry** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#379]
If you look at the official [Text fields](https://www.google.com/design/spec/components/text-fields.html) guidelines, the TextBox underline and floating hint are lighter than the foreground color (for the Light theme). Unfortunately, they don't say exactly what the opacity should be, but in all their images they're lighter. I've been able to fix the underline color by overriding the MaterialDesignTextBoxBorder (I set it to #20000000). But I haven't been able to customize the floating hint color since it is set to the body foreground color. Its opacity should probably be set to HintOpacity.

**Comment by EdonGashi** [comment]
Looking at SmartHint.xaml it seems like it already has its opacity set to HintOpacity, I think the visual state animation overrides it to 100% which makes it equal to the default body color:

```XML
<DoubleAnimation Duration="0"
    Storyboard.TargetName="FloatingHintTextBlock"
    Storyboard.TargetProperty="Opacity"
    To="1" />
```

**Comment by Keboo** [comment]
I believe this has been addressed in the latest 4.2.1 release
